selfdrive/car/hyundai/values.py

```diff
@@ -108,6 +108,7 @@ class CAR(StrEnum):
   # Kia
   KIA_FORTE = "KIA FORTE E 2018 & GT 2021"
   KIA_K5_2021 = "KIA K5 2021"
+  KIA_K5_HEV_2024 = "KIA K5 HYBRID 2024"
```

Comment:
Looks like this is the 5th gen refresh: https://en.wikipedia.org/wiki/Kia_K5#Facelift